Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -0,0 +1,336 @@
+            title: notification title
+            body: notification body
+        """
+        os.system("notify-send '{0}' '{1}' -u low".format(title,body))
+
+
+#Notification creation label
+#IN:
+#   title: Notification heading text
+#   body: Notification body text
+#   group: Notification group (for checking if we have this enabled)
+#   skip_checks: Whether or not we skips checks
+
+label display_notif(title, body, group=None, skip_checks=False):
+    #We only show notifications if:
+    #We are able to show notifs
+    #MAS isn't the active window
+    #User allows them
+    #And if the notification group is enabled
+
+    #OR if we skip checks
+    #NOTE: THIS IS TO ONLY BE USED FOR INTRODUCTORY PURPOSES
+
+    #First we want to create this location in the dict, but don't add an extra location if we're skipping checks
+    if persistent._mas_windowreacts_notif_filters.get(group) is None and not skip_checks:
+        $ persistent._mas_windowreacts_notif_filters[group] = False
+
+    if (
+            (mas_windowreacts.can_show_notifs
+            and ((renpy.windows and not mas_isFocused()) or not renpy.windows)
+            and mas_notifsEnabledForGroup(group))
+            or skip_checks
+        ):
+
+        #Make the notif
+        if (renpy.windows):
+            # The Windows way
+            play sound "mod_assets/sounds/effects/notif.wav"
+            $ tip.showWindow(renpy.substitute(title),renpy.substitute(body))
+
+            #We need the IDs of the notifs to delete them from the tray
+            $ destroy_list.append(tip.hwnd)
+
+        elif (renpy.macintosh):
+            # The macOS way
+            $ mas_tryShowNotificationOSX(renpy.substitute(title),renpy.substitute(body))
+
+        elif (renpy.linux):
+            # The Linux way
+            $ mas_tryShowNotificationLinux(renpy.substitute(title),renpy.substitute(body))
+    return
+
+
+#START: Window Reacts
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="monika_whatwatching",
+            category=['youtube'],
+            conditional="mas_notifsEnabledForGroup('Window Reactions')",
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label monika_whatwatching:
+    call display_notif(m_name,"What are you watching, [player]?",'Window Reactions')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="monika_lookingat",
+            category=['rule34', 'monika'],
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label monika_lookingat:
+    call display_notif(m_name, "Hey, [player]...what are you looking at?",'Window Reactions')
+
+    $ choice = random.randint(1,10)
+    if choice == 1:
+        $ queueEvent('monika_nsfw')
+
+    elif choice < 4:
+        show monika 1rsbssdlu
+
+    elif choice < 7:
+        show monika 2tuu
+
+    else:
+        show monika 2ttu
```

Comment:
seems that the last 2 options would just appear and then the idle loop would reset them back. might be worth pausing for a few seconds at least.